streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingPage.scala

```diff
@@ -145,6 +146,10 @@ private[ui] class StreamingPage(parent: StreamingTab)
   private val listener = parent.listener
   private val startTime = System.currentTimeMillis()
 
+  parent.parent.attachHandler(
+    StreamingApiRootResource.getServletHandler(parent.parent, listener, startTime)
```

Comment:
all right.
I supose i can make the change
but we need to decide where to record the startTime first
let me trace the code a little bit more.